lib/rollbar.rb

```diff
@@ -619,6 +605,33 @@ def send_failsafe(message, exception)
       failsafe_payload
     end
 
+    def failsafe_body(message, exception)
+      body = 'Failsafe from rollbar-gem. '
+      log_error "[Rollbar] Sending failsafe response due to #{message}."
+
+      if exception
+        begin
+          backtrace = exception.backtrace || []
+          nearest_frame = backtrace[0]
+
+          exception_info = exception.class.name
+          exception_info += " in #{nearest_frame}" if nearest_frame
+
+          body += "#{exception_info}: #{message}"
+
+          log_error "[Rollbar] #{body}"
+        rescue
+        end
+      else
+        begin
+          body += message.to_s
+        rescue
+        end
+      end
```

Comment:
@brianr this payload is passed through the dump method that will truncate it according to our strategies.

About the encoding. the message actually have:

- Our own part of message: 'Failsafe from rollbar-gem. '
- The exception class name. This doesn't have exception message value, it'll be safe.
- The nearest entry from backtrace. Here we could send rare chars to the API if the customer has some directory names with those chars, what will not be usual. But in this case I think is safer don't overuse the encoding logics we have so we don't add more potential risk to the safely report.